magefile.go

```diff
@@ -289,6 +289,16 @@ func Appveyor() error {
 	if err != nil {
 		return err
 	}
+	
+	err_x86 = Builder{
```

Comment:
Just change err_x86 = to err_x86 := 

arch should be 386 (see func Windowsx86 above).